README.md

```diff
@@ -33,6 +33,28 @@ Example of running cargo build on the kvm-ioctls crate:
     Finished release [optimized] target(s) in 5.63s
 ```
 
+## Testing Changes locally with the Container Image
+
+First build the rust-vmm container locally by running the commands
+
+```bash
+> cd rust-vmm-dev-container
+> ./docker.sh build
+```
+
+since this command will build a new docker image with tag latest version + 1
+and will alias it with "latest" tag, when testing the container use the latest
+tag that will be published with your PR to be sure that the changes introduced
+by your PR to the CI works correctly before pusing it upstream.
+
+Example of how to test the container on your localhost with tag v37:
+
+```bash
+> docker run --device=/dev/kvm -it --rm --security-opt=seccomp=unconfined \
+--volume $(pwd):/path/to/workdir --workdir /path/to/workdir \
+--privileged rustvmm/dev:v37
```

Comment:
Should we give an example of what `workdir` should be?

IIRC `--security-opt=seccomp=unconfined` was required only by `podman`. Did you test with `docker` or `podman-docker`?

Reply:
yeah I think it is good to give example. I tested with podman.